runelite-client/src/main/java/net/runelite/client/plugins/ibansstaff/IbansStaffConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018, JerwuQu <marcus@ramse.se>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.ibansstaff;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "ibansStaff",
+	name = "Iban's Staff",
+	description = "Configuration for the Iban's Staff plugin"
+)
+public interface IbansStaffConfig extends Config
+{
+	@ConfigItem(
+		keyName = "alwaysShow",
+		name = "Always show charges",
+		description = "Always displays amount of Iban's Staff charges"
+	)
+	default boolean alwaysShow()
```

Comment:
This should probably not be an option, and just show the infobox when staff is equipped, otherwise show counter on the staff itself (it would be shown also when equipped i guess, see JewelleryCount). Just adding boolean for even showing the infobox should suffice.

Reply:
Having the option be "Display infobox when equipped" and having a counter in the inventory sounds like the best solution. Will fix as soon as the discussed JewelleryCount changes have been made.